### DISCUSSION TITLE
Next TSC Meeting - Thu 2024-09-12 @ 1300 UTC

### DISCUSSION THREAD

**Discussion by planetf1** [pq-code-package/tsc#99] [Announcements]
Our next TSC meeting is this Thursday

**Comment by planetf1** [comment]
I'd particularly like to discuss
mlkem-c-generic : update on progress
additional algorithms we could source/manage - ML-DSA, MLH-DSA & potentially others (see minutes)
getting to a point where we can supply relevant algos to liboqs

**Comment by franziskuskiefer** [comment]
I have a last minute conflict unfortunately.
